docs/source-2.0/aws/aws-endpoints.rst

```diff
@@ -0,0 +1,375 @@
+.. _aws-endpoints:
+
+===============================
+AWS Declarative Endpoint Traits
+===============================
+
+This document defines AWS Declarative endpoint traits.
+
+
+.. smithy-trait:: aws.endpoints#endpointsModifier
+.. _aws.endpoints#endpointsModifier-trait:
+
+-----------------------------------------
+``aws.endpoints#endpointsModifier`` trait
```

Comment:
Can the summaries for traits that apply this trait be updated to lead with "An endpoints modifier trait that...", where `endpoints modifier trait` links to this section?

Reply:
done!